### PR TITLE
support Admonition, Tabs and @status-im/icons in md

### PR DESCRIPTION
**Pull request by felicio**

### CODE REVIEW

File: src/components/icons/desktop/index.ts
Comment (felicio) [inline src/components/icons/desktop/index.ts:1]: fixme!: desktop icons are rendered without colored line/outlines even if explicitly set

File: contentlayer.config.ts [hunk 1]
```diff
@@ -134,40 +133,6 @@ function getPathSegments(filePath: string) {
...
-        // @ts-expect-error TODO
-        node.attributes = [
-          { type: 'mdxJsxAttribute', name: 'type', value: type },
-          // @ts-expect-error TODO
-          { type: 'mdxJsxAttribute', name: 'title', value: node.label },
-        ]
-      }
-    })
-  }
-}
```
Comment (prichodko) [inline contentlayer.config.ts:169]: 🥲

File: src/pages/help/[...slug].tsx [hunk 2]
```diff
@@ -305,11 +351,33 @@ const components = {
...
+        <Tabs.List {...props} />
+      </div>
+    )
+  },
+  TabsTrigger: (props: any) => {
+    return (
+      <Tabs.Trigger {...props}>{props.children.props.children}</Tabs.Trigger>
+    )
+  },
+  ...inconComponents,
```
Comment (prichodko) [inline src/pages/help/[...slug].tsx:379]: typo

**Review by prichodko — APPROVED** [review]

File: src/pages/help/[...slug].tsx (hunk 2 above)
Comment (felicio) [reply src/pages/help/[...slug].tsx]: https://github.com/status-im/status-website/pull/13/commits/67b41f6f57ecfc0fdfa6f55efa3ac0d64542f166

**Pull request merged**